setup.py

```diff
@@ -13,6 +13,6 @@
     long_description="Monufacture is a factory framework with an API designed to make " +
                      "it as easy as possible to generate valid test data in MongoDB. " +
                      "Inspired by the excellent factory_girl Ruby Gem.",
-    install_requires=['pymongo<3.0.0', 'pytz'],
+    install_requires=['pymongo>=3.0.0,<4.0.0', 'pytz'],
```

Comment:
Exact version pegs are for applications. This is a library. Libraries need to be flexible in the versions they support.